File: GraphPlan (sgware)/src/edu/uno/ai/planning/gp/SubgraphRoot.java
package edu.uno.ai.planning.gp;

import edu.uno.ai.planning.Planner;
import edu.uno.ai.planning.pg.PlanGraph;

public class SubgraphRoot extends SubgraphNode {

	int limit = Planner.NO_NODE_LIMIT;
	
	SubgraphRoot(PlanGraph graph) {
		super(graph);
	}
	
	void setNodeLimit(int limit) {
		this.limit = limit;
	}
}

File: UnitySokoban/Assets/Scripts/Planning/GraphPlanSGW/SubgraphRoot.cs
using PlanGraphSGW;
using Planning;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace GraphPlanSGW
{
    public class SubgraphRoot : SubgraphNode
    {
        internal int limit = Planner<Search>.NO_NODE_LIMIT;

        internal SubgraphRoot(PlanGraph graph)
            : base(graph)
        {
        }

        internal void setNodeLimit(int limit)
        {
            this.limit = limit;
        }
    }

}
